src/Controller/Admin/MemberCrud.php

```diff
@@ -107,6 +114,55 @@ public function cancelMembership(AdminContext      $adminContext,
         return $this->redirect($redirectUrl);
     }
 
+    public function convertMemberToSupportMember(AdminContext $adminContext, AdminUrlGenerator $adminUrlGenerator, EntityManagerInterface $entityManager): Response
+    {
+        /**
+         * @var Member
+         */
+        $member = $adminContext->getEntity()->getInstance();
+
+        // check if current user is allowed and the selected member is allowed to be removed and converted
+        $redirectUrl = $adminUrlGenerator
+            ->setController(self::class)
+            ->setAction(Crud::PAGE_EDIT)
+            ->setEntityId($member->getId())
+            ->generateUrl();
+
+        $supportMember = new SupportMember();
+        $supportMember->setFirstName($member->getFirstName());
+        $supportMember->setLastName($member->getLastName());
+        $supportMember->setEmail($member->getEmail());
+        $supportMember->setPhone($member->getPhone());
+        $supportMember->setIban($member->getIban());
+        $supportMember->setAddress($member->getAddress());
+        $supportMember->setCity($member->getCity());
+        $supportMember->setPostCode($member->getPostCode());
+        $supportMember->setCountry($member->getCountry());
+        $supportMember->setDateOfBirth($member->getDateOfBirth());
+        $supportMember->setRegistrationTime(new DateTime()); // now seems more logically
+
+        // Should this become a new one, because i assume message is different, don't know where it is set
```

Comment:
Goede vraag, steunleden betalen vaak andere contributie en ik weet niet zeker of we het bedrag kunnen aanpassen zonder opnieuw mandaat te vragen.

Reply:
Een mollie subscription heeft een methode om het bedrag te updaten. zie: https://github.com/roodjong/mijnrood/blob/79c9e7690f0f77581839f6dd8e31844cc1a408cb/src/Controller/Admin/SupportMemberCrud.php#L209
Maar ik weet niet of het zomaar mag